src/handlers/testing/exec/components/progress/index.tsx

```diff
@@ -232,42 +226,43 @@ const App = (props: AppProps) => {
       });
     };
 
-    const uncaughtErrorListener = (error: UncaughtError) => {
```

Comment:
why was this listener removed?

Reply:
decided to drop it because we were only using it to make the dots white instead of gray for uncaught errors, which didn't seem like a good enough reason to pipe it thru to the ink app